plugins/obs-ffmpeg/obs-ffmpeg-hls-mux.c

```diff
@@ -0,0 +1,339 @@
+#include "obs-ffmpeg-mux.h"
```

Comment:
Automatic checks show that plugins/obs-ffmpeg/obs-ffmpeg-hls-mux.c is not formatted properly.